### PR TITLE
doc: generate v1.12.3

### PR DESCRIPTION
**Pull request by lantoli**
## Description

Ticket: [INTMDB-1246](https://jira.mongodb.org/browse/INTMDB-1246)

This PR is to generate changelog for v1.12.3.

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue). Please, add the "bug" label to the PR.
- [ ] New feature (non-breaking change which adds functionality). Please, add the "enhancement" label to the PR.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected). Please, add the "breaking change" label to the PR.
- [ ] This change requires a documentation update
- [X] Documentation fix/enhancement

## Required Checklist:

- [X] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [X] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [X] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [X] I have added any necessary documentation (if appropriate)
- [X] I have run make fmt and formatted my code

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
...
+- Provides a `mongodbatlas_privatelink_endpoint` by region when using data source `mongodbatlas_privatelink_endpoint`  [\#1525](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1525)
+- Error message: Either Atlas Programmatic API Keys or AWS Secrets Manager attributes must be set [\#1483](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1483)
+- Unable to update Slack `alert_configurations` via Oauth integration  [\#1074](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1074)
+
+**Internal Improvements**
+
+- Disable `assume_role` [\#1583](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1583) ([andreaangiolillo](https://github.com/andreaangiolillo))
+- Fixes import documentation for `project_api_key` resource [\#1582](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1582) ([lantoli](https://github.com/lantoli))
+- Fixes attributes paused, `version_release_system` and tags in advanced cluster resource [\#1581](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1581) ([lantoli](https://github.com/lantoli))
```
Comment (marcosuma) [inline CHANGELOG.md]:
```suggestion
- Fixes attributes `paused`, `version_release_system` and `tags` in advanced cluster resource [\#1581](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1581) ([lantoli](https://github.com/lantoli))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+
+- `replication_specs` do not support deep diff [\#1544](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1544)
+- There is no `team_id` argument/attribute reference in `mongodbatlas_project_invitation` resource block. [\#1535](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1535)
+- Provides a `mongodbatlas_privatelink_endpoint` by region when using data source `mongodbatlas_privatelink_endpoint`  [\#1525](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1525)
+- Error message: Either Atlas Programmatic API Keys or AWS Secrets Manager attributes must be set [\#1483](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1483)
+- Unable to update Slack `alert_configurations` via Oauth integration  [\#1074](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1074)
+
+**Internal Improvements**
+
+- Disable `assume_role` [\#1583](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1583) ([andreaangiolillo](https://github.com/andreaangiolillo))
```
Comment (marcosuma) [inline CHANGELOG.md]:
```suggestion
- Disable `assume_role` acceptance test workflow [\#1583](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1583) ([andreaangiolillo](https://github.com/andreaangiolillo))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+- Fixes import documentation for `project_api_key` resource [\#1582](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1582) ([lantoli](https://github.com/lantoli))
+- Fixes attributes paused, `version_release_system` and tags in advanced cluster resource [\#1581](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1581) ([lantoli](https://github.com/lantoli))
+- Updates run condition in migration tests github action [\#1580](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1580) ([maastha](https://github.com/maastha))
+- Does not delete project for trigger acctest. [\#1573](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1573) ([marcosuma](https://github.com/marcosuma))
+- Updates migration tests to run separately and use last released version of provider for plan checks [\#1565](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1565) ([maastha](https://github.com/maastha))
+- Fixes region and aws account to be used for trigger acc test [\#1558](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1558) ([marcosuma](https://github.com/marcosuma))
```
Comment (marcosuma) [inline CHANGELOG.md]:
```suggestion
- Fixes aws region and aws account to be used for trigger acceptance test [\#1558](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1558) ([marcosuma](https://github.com/marcosuma))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,59 @@
...
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.12.2...v1.12.3)
+
+**Enhancements**
+
+- Adds `acceptDataRisksAndForceReplicaSetReconfig` parameter in `mongodbatlas_cluster` and `mongodbatlas_advanced_cluster` [\#1575](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1575) ([lantoli](https://github.com/lantoli))
+
+**Bug Fixes**
+
+- Overrides to attribute behavior for resource elems. [\#1572](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1572) ([marcosuma](https://github.com/marcosuma))
+- Fixes 'computed' and 'default' usage based on the documentation [\#1564](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1564) ([marcosuma](https://github.com/marcosuma))
```
Comment (jwilliams-mongo) [inline CHANGELOG.md]: minor suggestion to keep formatting consistent
```suggestion
- Fixes `computed` and `default` usage based on the documentation [\#1564](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1564) ([marcosuma](https://github.com/marcosuma))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,59 @@
...
+
+- `replication_specs` do not support deep diff [\#1544](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1544)
+- There is no `team_id` argument/attribute reference in `mongodbatlas_project_invitation` resource block. [\#1535](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1535)
+- Provides a `mongodbatlas_privatelink_endpoint` by region when using data source `mongodbatlas_privatelink_endpoint`  [\#1525](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1525)
+- Error message: Either Atlas Programmatic API Keys or AWS Secrets Manager attributes must be set [\#1483](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1483)
+- Unable to update Slack `alert_configurations` via Oauth integration  [\#1074](https://github.com/mongodb/terraform-provider-mongodbatlas/issues/1074)
+
+**Internal Improvements**
+
+- Disable `assume_role` acceptance test workflow [\#1583](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1583) ([andreaangiolillo](https://github.com/andreaangiolillo))
```
Comment (jwilliams-mongo) [inline CHANGELOG.md]: making this consistent with the rest of the items
```suggestion
- Disables `assume_role` acceptance test workflow [\#1583](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1583) ([andreaangiolillo](https://github.com/andreaangiolillo))
```

**Review by jwilliams-mongo — APPROVED** [review]
@lantoli LGTM % a few minor suggestions for consistency

**Review by Zuhairahmed — APPROVED** [review]
Added few small commits directly, otherwise LGTM. nice job @lantoli !  feel free to merge + release

File: CHANGELOG.md (hunk 4 above)
Comment (lantoli) [reply CHANGELOG.md]: done, thx

**Comment by lantoli** [comment]
> @lantoli LGTM % a few minor suggestions for consistency

thanks!

**Pull request merged**